store/index.ts

```diff
@@ -78,6 +80,7 @@ export const actions: ActionTree<RootState, RootState> = {
     promises.push(dispatch('backdrop/clearAllBackdrop', { root: true }));
     promises.push(dispatch('deviceProfile/clearDeviceProfile', { root: true }));
     promises.push(dispatch('displayPreferences/resetState', { root: true }));
+    promises.push(dispatch('items/clearState', { root: true }));
```

Comment:
Maybe we can except this store, so if an user logs out the data it's loaded instantly for the next user (if the user it's in the same server)?

Although I guess it's better to keep it for consistency.

Reply:
That's from the items store PR :)

Main issue with keeping items between sessions, though, is that they might have UserData stuck to them, which isn't good.